src/shared/components/action-buttons/Align.tsx

```diff
@@ -1,39 +1,95 @@
+import { Button, Dropdown } from 'franklin-sites';
+
 import ToolsButton from './ToolsButton';
 
 import { fromCleanMapper } from '../../utils/getIdKeyForNamespace';
+import { pluralise } from '../../utils/utils';
 
 import { Location } from '../../../app/config/urls';
+import { JobTypes } from '../../../tools/types/toolsJobTypes';
+import { PublicServerParameters } from '../../../tools/types/toolsServerParameters';
 
 const ALIGN_LIMIT = 100;
+const isDisabled = (n: number) => n <= 1 || n > ALIGN_LIMIT;
+
+const getTitle = (n: number) => {
+  if (n > 1) {
+    if (n > ALIGN_LIMIT) {
+      return `Please select a maximum of ${ALIGN_LIMIT} entries to run an Align job`;
+    }
+    return `Run an Align job against ${n} entries`;
+  }
+  return 'Select at least 2 entries to run an Align job';
+};
 
 type AlignButtonProps = {
   selectedEntries: string[];
   textSuffix?: string;
+  inputParamsData?: PublicServerParameters[JobTypes];
 };
 
-const AlignButton = ({ selectedEntries, textSuffix }: AlignButtonProps) => {
+const AlignButton = ({
+  selectedEntries,
+  textSuffix,
+  inputParamsData,
+}: AlignButtonProps) => {
   const cleanedSelectedEntries = Array.from(
     new Set(selectedEntries.map(fromCleanMapper))
   );
 
   const n = cleanedSelectedEntries.length;
 
-  const disabled = n <= 1 || n > ALIGN_LIMIT;
+  const sequence =
+    inputParamsData &&
+    'sequence' in inputParamsData &&
+    inputParamsData.sequence;
 
-  let title = 'Select at least 2 entries to run an Align job';
-  if (n > 1) {
-    if (n > ALIGN_LIMIT) {
-      title = `Please select a maximum of ${ALIGN_LIMIT} entries to run an Align job`;
-    } else {
-      title = `Run an Align job against ${n} entries`;
-    }
+  if (sequence) {
+    return (
+      <Dropdown
+        visibleElement={
+          <Button
+            // If both buttons within were to be disabled
+            disabled={isDisabled(n) && isDisabled(n + 1)}
+            variant="tertiary"
+            title={`Select from 2 to ${ALIGN_LIMIT} entries to run an Align job`}
```

Comment:
```suggestion
            title={`Select from 1 to ${ALIGN_LIMIT} entries to run an Align job`}
```
As we can align against the query